about.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="index.css" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Lato"
+      rel="stylesheet"
+    />
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <script src="index.js"></script>
+
+    <title>Capstone Project - Concert Page</title>
+  </head>
+  <body>
+    <header>
+      <div class="upper-header">
+        <ul class="list1">
+          <li><img src="./assets/fb icon grey.png" alt="facebook-link" class="header-top-img"/></li>
+          <li><img src="./assets/twitter icon grey.png" alt="twitter-link" class="header-top-img"/></li>
+          <li><p>English</p></li>
+          <li><p>My Page</p></li>
+          <li><p>Logout</p></li>
+        </ul>
+      </div>
+      <nav>
+        <a href="index.html"><img src="./assets/Logo.webp" alt="logo" id="logo"/></a>
+        <ul class="list2">
+          <li class="desktop-nav"><a href="about.html" style="color: #ec5242;">About</a></li>
+          <li class="desktop-nav"><a href="#">Program</a></li>
+          <li class="desktop-nav"><a href="#">Join</a></li>
+          <li class="desktop-nav"><a href="#">Sponsor</a></li>
+          <li class="desktop-nav"><a href="#">News</a></li>
+          <li class="hamburger">
+            <img src="./assets/hamburger.png" alt="hamburger" id="hamburger"/>
+          </li>
+        </ul>
+      </nav>
+      <div class="mobile-menu">
+        <ul class="list3">
+          <li id="close" class="mobile-nav">
+            <img src="./assets/Icon.png" alt="close" id="close-button"/>
+          </li>
+          <li class="mobile-nav"><a href="about.html">About</a></li>
+          <li class="mobile-nav"><a href="#">Program</a></li>
+          <li class="mobile-nav"><a href="#">Join</a></li>
+          <li class="mobile-nav"><a href="#">Sponsor</a></li>
+          <li class="mobile-nav"><a href="#">News</a></li>
+        </ul>
+      </div>
+    </header>
+    <section class="home">
+      <h3>About...</h3>
+      <img src="assets/aboutLOGO.png" alt="program logo" id="about-logo" />
```

Comment:
- [ ] Great job with the styling of this headline, however it's required that you put a header here and not an image, to follow the design guidelines 👍 Kindly add a header text with the corresponding `font-family`, `font-weight` and `font-size`

## Your design:
![image](https://user-images.githubusercontent.com/50721479/204087840-9521d76d-d28a-412a-9fb4-7dedbce89a9e.png)

## Guidelines:
![image](https://user-images.githubusercontent.com/50721479/204087848-8cf0451a-3019-4446-b8f0-29bdfdeeacc0.png)